docs/contact.html

```diff
@@ -68,25 +65,29 @@ <h2>
                 <legend>Do you play games?</legend>
               
                 <div>
-                  <input type="radio" id="Yes" name="Yes" value="Yes" checked />
+                  <input type="radio" id="Yes" name="my_radio" value="Yes" >
                   <label>Yes</label>
                 </div>
               
                 <div>
-                  <input type="radio" id="No" name="No" value="No" />
+                  <input type="radio" id="No" name="my_radio" value="No" >
                   <label>No</label>
                 </div>
               
                 <div>
-                  <input type="radio" id="Prefer Not To Say" name="Prefer Not To Say" value="Prefer Not To Say" />
+                  <input type="radio" id="Prefer Not To Say" name="my_radio" value="Prefer Not To Say" >
                   <label ">Prefer Not To Say</label>
                 </div>
+              
               </fieldset>
-
           </div>
+          <div>
+                      </div>
           <div class="formmat" >
-            <input type="reset" value="Reset" /> <a href="mailto:someone@example.com?Subject=Hello%20again" target="_top">
-            Submit</a>
+            <input type="reset" value="Reset" /> <input type="submit" id="submit" name="submit" onclick="checkName(document.myform.yourname)">
```

Comment:
The `click` event can be used to handle button clicks, but it doesn't actually handle form submission.  Use the `submit` event on `<form>` elements for that instead